### PR TITLE
add elastic search support

### PR DESCRIPTION
**Pull request by dvictory**

### CODE REVIEW

**Comment by jspies** [comment]
looks good. does @sdrioux know you did this? He wanted to add the multi order by

:shipit:

File: lib/sapine.rb [hunk 1]
```diff
@@ -23,16 +23,56 @@ def index_options(chain)
...
+        order[value[1..-1]] = 'desc'
+
+      elsif value.starts_with? "+"
+        order[value[1..-1]] = 'asc'
+
+      else
+        order[value] = 'asc'
+      end
+    end
+    order
```
Comment (sdrioux) [inline lib/sapine.rb:63]: :+1: , correct me if I"m wrong, but this means we can pass either { order_by: "-name" }, or { order_by: ["-name", "+id"] }, and both will work?

**Comment by sdrioux** [comment]
:ship:

**Comment by texasjusticar** [comment]
:+1:

File: lib/sapine.rb (hunk 1 above)
Comment (dvictory) [inline lib/sapine.rb:63]: if you do that you will get a return value of {name: 'desc', id: 'asc'}

**Pull request merged**